src/constraints/outage_constraints.jl

```diff
@@ -342,8 +349,32 @@ function add_MG_storage_dispatch_constraints(m,p)
             m[:dvMGDischargeFromStorage][s, tz, ts] <= p.s.storage.attr["ElectricStorage"].max_kw * m[:binMGStorageUsed]
         )
     end
-end
 
+    # Future development could make this an option by adding bool inputs an updating load balance/battery dispatch constraints to include these flows
+    if "ElectricStorage" in p.s.storage.types.dc_coupled
+        # Don't let AC coupled elec techs charge DC coupled battery. 
+        for ts in 1:p.s.site.min_resil_time_steps
+            for tz in p.s.electric_utility.outage_start_time_steps
+                for s in p.s.electric_utility.scenarios
+                    for t in p.techs.ac_coupled_with_storage
+                        fix(m[:dvMGProductionToStorage][t, s, tz, ts], 0.0, force=true)
+                    end
+                end
+            end
+        end
+    else
```

Comment:
would this ever be reached, currently? For this else statement to activate you'd need to have ElectricStorage dc_coupled be false, but have the dc_coupled_with_storage tech set be not empty?